soccer/vision/kick/detector/FastKickDetector.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <deque>
+
+#include <Geometry2d/Point.hpp>
+#include <Utils.hpp>
+
+#include "vision/ball/WorldBall.hpp"
+#include "vision/robot/WorldRobot.hpp"
+#include "vision/kick/KickEvent.hpp"
+#include "vision/kick/VisionState.hpp"
+
+class FastKickDetector {
```

Comment:
It would be good to get a quick explanation of what this does. Does it detect kicks quickly or does it detect kicks that give the ball a high velocity. Same for the SlowKickDetector